Documentation/security/threat-model.rst

```diff
@@ -0,0 +1,751 @@
+|                  |                                                   |
+|                  | -  In overlay mode, the attacker could spoof      |
+|                  |    traffic that appears to come from any          |
+|                  |    workload via spoofing identities.              |
+|                  |                                                   |
+|                  | -  If the attacker was able to control the        |
+|                  |    ipcache through another attack (for            |
+|                  |    example, by either being a successful          |
+|                  |    :ref:`API Server attacker                      |
+|                  |    <kubernetes-api-server-attacker>`              |
+|                  |    or :ref:`key-value store attacker              |
+|                  |    <kv-store-attacker>`                           |
+|                  |    they could also spoof traffic by spoofing      |
+|                  |    pod identities.                                |
+|                  |                                                   |
+|                  | Denial of service could occur depending on the    |
+|                  | behavior of the attacker.                         |
++------------------+---------------------------------------------------+
+| Observability    | Denial of service could occur depending on the    |
+| data             | behavior of the attacker.                         |
+|                  | |br| |br|                                         |
+|                  | Information leakage could occur via an attacker   |
+|                  | scraping Cilium or Hubble Prometheus metrics,     |
+|                  | depending on the specific metrics enabled.        |
++------------------+---------------------------------------------------+
+
+Recommended Controls
+^^^^^^^^^^^^^^^^^^^^
+
+:ref:`gsg_encryption` should be configured to ensure the confidentiality of
+communication between workloads.
+
+.. _kubernetes-api-server-attacker:
+
+Kubernetes API Server Attacker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This type of attack could be carried out by any user or code with
+network access to the Kubernetes API server and credentials that allow
+Kubernetes API requests. Such permissions would allow the user to read
+or manipulate the API server state (for example by changing CRDs).
+
+This is not a precise term, as different RBAC policies could grant many
+different types of privilege. This section is intended to cover any
+attack that might be exposed via Kubernetes API server access, whether
+it is “full” API server access, or even limited API server access (for
+example, access gained by compromising credentials used by Cilium).
+
+`The Kubernetes documentation`_ offers more information about configuring RBAC.
+
+.. _The Kubernetes documentation: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
```

Comment:
I think that this reference should be included, but it seems like it's already included below under `Recommended controls`. Do we need it here? The sentence reads a bit odd all by itself here.